src/test/java/seedu/address/commons/util/StringUtilTest.java

```diff
@@ -109,7 +109,7 @@ public void containsWordIgnoreCase_validInputs_correctResult() {
         assertFalse(StringUtil.containsWordIgnoreCase("    ", "123"));
 
```

Comment:
Should adjust the test docstring above also to reflect that it checks for partial word matches

Reply:
Alright! Thanks, will do,